### DISCUSSION TITLE
Question in my head 💬

### DISCUSSION THREAD

**Discussion by SebastienXia** [community/community#110673] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I asked myself the question: "Is it still useful to learn programming languages ​​and all kinds of things in this field in view of the development of AI. **What will the future** of certain IT professions look like? ?”

**Comment by 4chref** [comment]
Absolutely, learning programming languages and gaining skills in various fields of IT remains highly valuable, even with the rapid advancement of AI. While AI technologies are becoming more sophisticated and capable of automating certain tasks, they still rely heavily on human input for development, maintenance, and oversight.

**Comment by gil-son** [comment]
Improve your skills and combine it with AI to optimize yourself.

Use AI in your favor.
